src/styles/index.scss

```diff
@@ -15,3 +15,9 @@
 .usa-search [type='search']::-webkit-search-cancel-button {
   display: none;
 }
+
+// This just allows buttons as well as links in the LanguageSelector
+// It can be removed when https://github.com/uswds/uswds/issues/5409 is fixed
+.usa-language__submenu .usa-language__submenu-item button {
+  @extend a;
```

Comment:
```suggestion
    color: color('white');
  display: block;
  line-height: line-height($theme-navigation-font-family, 3);
  padding: 0;
  padding: units(1);
  text-decoration: none;
  width: 100%;  /* this is something that was actually missing when we were doing the extend because button widths hve different default widths than block element links */

  &:focus {
    outline-offset: units('neg-05');
  }

  &:hover {
    color: color('white');
    text-decoration: underline;
  }
```

Reply:
this just grips the scss [right out of the package](https://github.com/uswds/uswds/blob/develop/packages/usa-language-selector/src/styles/_usa-language-selector.scss), replicating the @extend output.

only advantage of this over using @extend is it stops us from overriding lint rules, and it makes sure the focus indicator behaves the same with with width assignment I added, but you could also add that using the @extend.

I would defer to the app eng folks on how they feel about overriding lint rules, but if you are looking to get this going this is how I'd get around it.

Reply:
thanks! committed that

Reply:
hm, we don't seem to be using utilities _anywhere_, and I couldn't get it to work, so I've just copied the compiled styles over here for now.